test/common/Error.spec.ts

```diff
@@ -158,7 +158,7 @@ describe('ZoneAwareError', () => {
   });
 
   it('should show zone names in stack frames and remove extra frames', () => {
-    const rootZone = getRootZone();
+    const rootZone = Zone.root;
```

Comment:
I have added the test case of Error('foo'), and because Error without new will cause different stack frames, so I also add detect logic in zone.ts.